### DISCUSSION TITLE
Find 15 words related to "Force"

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#606] [Galaxy Trivia] [answered]
<img width="218" alt="image" src="https://user-images.githubusercontent.com/5120290/233685073-651182e8-499b-4d37-8c00-2b133772a626.png">

Directions: ➡️⬇️⬅️⬆️

**Comment by umlaeute** [comment, marked as answer]
1. friction :arrow_right:
2. pull  :arrow_left:
3. speed :arrow_right:
4.  force  :arrow_right:
5. thrust  :arrow_left:
6. distance  :arrow_left:
7.  newton :arrow_right:
8. newtonmeter :arrow_left:
10.  balance :arrow_right:
11.  resistance  :arrow_left:
12.  dynamic :arrow_left:
13.  mass  :arrow_up:
14.  low  :arrow_up:
15.  measure  :arrow_up:
16. meter  :arrow_left:

**Reply by Taky0nDon** [reply]
finally, something on my level
1. friction ➡️
2.  meters⬅️
3.  speed➡️
4. newtons➡️
6.  balanced➡️
7. balanced ⬅️
8.  pull⬅️
9. forces ➡️
10.  Newton meters⬅️
11.  resistance⬅️
12.  mass⬆️
13. bam ⬇️
14. measure ⬆️
15.  distance ⬅️

**Reply by ParanoidUser** [reply]
Good job guys! @umlaeute, made it an inch closer.

1. NEWTONMETERS ⬅️
2. MEASURE ⬆️
3. DISTANCE ⬅️
4. FRICTION ➡️
5. BALANCED ⬅️
6. AERODYNAMIC ⬅️
7. RESISTANCE ⬅️
8. NEWTONS ➡️
9. SPEED ➡️
10. ISSACNEWTON ➡️
11. MASS ⬆️
12. PULL ⬅️
13. UNBALANCED ➡️
14. UPTHRUST ⬅️
15. FORCES ➡️

**Comment by thuongtruong1009** [comment]
Here are 15 words related to "Force" based on the image:

Push
Pull
Tension
Compression
Thrust
Gravitation
Inertia
Acceleration
Friction
Velocity
Momentum
Impact
Load
Weight
Pressure
